Classes/Utility/AttributeUtility.php

```diff
@@ -41,6 +41,58 @@ public static function findAttribute(int $attributeId, string $selectFields = '*
         );
     }
 
+    /**
+     * Returns all attribute id's for corresponding product.
+     *
+     * @param int $productId
+     * @param string $selectFields
+     * @return array|null
+     */
+    public static function findAllAttributesForProduct(
+        int $productId,
+        array $attributeIds = [],
+        array $attributeIdentifiers = [],
+        string $selectFields = '*'
+    ): ?array {
```

Comment:
Reflecting reality. ;-)

```suggestion
    ): array {
```